ui/app/AppLayouts/Communities/CommunitiesPortalLayout.qml

```diff
@@ -137,14 +137,18 @@ StatusSectionLayout {
                     onClicked: Global.importCommunityPopupRequested()
                 }
 
-                StatusButton {
-                    id: createBtn
-                    objectName: "createCommunityButton"
-                    Layout.preferredHeight: 38
-                    verticalPadding: 0
-                    text: qsTr("Import existing community from Discord")
-                    onClicked: {
-                        Global.createCommunityPopupRequested(true /*isDiscordImport*/)
+                Loader {
+                    Layout.preferredHeight: active ? 38 : 0
+                    active: communitiesStore.createCommunityEnabled || communitiesStore.testEnvironment
```

Comment:
Showing it on `testEnvironment: true` is important, otherwise a lot of tests would fail.

Reply:
Maybe write this in the code as comment as well?